Watch Extension/AddFeedingView.swift

```diff
@@ -0,0 +1,106 @@
+                    })
+                Spacer()
+            }
+            Spacer()
+            HStack {
+                Spacer()
+                Image(systemName: "b.circle.fill")
+                    .font(.system(size: 55))
+                    .scaleEffect(self.intent.isAdding ? 0.5 : 1)
+                    .gesture(TapGesture().onEnded {
+                        withAnimation(.spring(response: 0.35, dampingFraction: 0.65)) { self.intent = .bottle }
+                    })
+                Spacer()
+            }
+            Spacer()
+        }
+    }
+}
+
+// TODO: change this to be backed by an int
+enum AddingIntent: String {
+    // TODO: I may be able to use this enum from the main app
+    case nursing, pumping, bottle, none
+
+    var isAdding: Bool {
+        switch self {
+        case .nursing, .pumping, .bottle: return true
+        case .none: return false
+        }
+    }
+}
+
+struct AddFeedingView: View {
+    @Binding var isShowingSheet: Bool
+    @State private var intent: AddingIntent = .none
+
+    var body: some View {
+        ZStack {
+            FeedingOptionsView(intent: $intent)
+            GeometryReader { metrics in
+                ZStack {
+                    Rectangle()
+                        .opacity(self.intent.isAdding ? 0.5 : 0)
+                        .foregroundColor(Color.black)
+                        .gesture(TapGesture().onEnded {
+                            // TODO: not a fan of duplicating this
+                            withAnimation(.spring(response: 0.35, dampingFraction: 0.65)) { self.intent = .none }
+                        })
+
+                    Image(systemName: "l.circle")
+                        .font(.system(size: 55))
+                        // TODO: These numbers are not exact, figure this out better
+                        .offset(x: self.intent.isAdding ? -45 : -metrics.size.width * 0.5 - 50)
+                        .gesture(TapGesture().onEnded {
+                            WCSession.default.sendMessage(["feedingType": "\(self.intent.rawValue)", "side": "left"], replyHandler: { reply in
+                                print(reply)
+                            }) { e in
+                                print("Error sending the message: \(e.localizedDescription)")
+                            }
+                            self.isShowingSheet = false
+                        })
+
+                    Image(systemName: "r.circle")
+                        .font(.system(size: 55))
+                        .offset(x: self.intent.isAdding ? 45 : metrics.size.width * 0.5 + 50)
+                        .gesture(TapGesture().onEnded {
+                            WCSession.default.sendMessage(["feedingType": "\(self.intent.rawValue)", "side": "right"], replyHandler: { reply in
+                                print(reply)
+                            }) { e in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c49e444d61917df61aa42d9cccb11a46857e6fb9/Watch Extension/AddFeedingView.swift#L91"></span>Trailing closure syntax should not be used when passing more than one closure argument.<br />
<code>multiple_closures_with_trailing_closure</code> <code>AddFeedingView.swift:91</code></td>
    </tr>
  </tbody>
</table>